### PR TITLE
Fix Github Light and Pitaya Smoothie contrast color bugs

### LINKED ISSUE #59: Miscellaneous improvements needed on themes
## GitHub light

- [x] The active tab colour is not noticeable (need to swap to a different colour)
plugin_json_-_JupyterLab

Compared to GitHub dark
plugin_json_-_JupyterLab

- [x] Hover background (menu) does not have enough contrast against the background colour
RELEASE_md__2__-_JupyterLab

- [x] Colours in the kernel/tab manager sidebar have gone missing - all is grey now, which makes it look like it is all text and lacks some of the "warning" type contexts
RELEASE_md__2__-_JupyterLab

Compared to JupyterLab light
RELEASE_md__2__-_JupyterLab

- [x] There is no active indicator on the `search` bar

RELEASE_md__2__-_JupyterLab

Compared to JupyterLab light

RELEASE_md__2__-_JupyterLab

- [x] Toggle background is not noticeable
RELEASE_md__2__-_JupyterLab

## GitHub dark

- [x] Selected file in File browser has no background (or is the same as the overall sidebar background)

JupyterLab

Compared to JupyterLab light
JupyterLab

- [x] Some icons are rendered in black, which does not have enough contrast against the background (in general the colours of the icons are all very similar, same on the light theme)

binder_-_JupyterLab

Compared to JupyterLab dark
binder_-_JupyterLab

- [ ] Hover background (menu) does not have enough contrast against the background colour
RELEASE_md__2__-_JupyterLab

- [x] Colours in the kernel/tab manager sidebar have gone missing - all is grey now, which makes it look like it is all text and lacks some of the "warning" type contexts
RELEASE_md__2__-_JupyterLab

- [x] There is no active indicator on the `search` bar
RELEASE_md__2__-_JupyterLab

## Pitaya Smoothie

- [ ] Some icons are rendered in black, which does not have enough contrast against the background
binder_-_JupyterLab

- [ ] There is no active indicator on the `search` bar
RELEASE_md__2__-_JupyterLab

- [ ] When changing the theme to Pitaya Smoothie, the bottom bar disappears
RELEASE_md__2__-_JupyterLab

Compared to GH dark
RELEASE_md__2__-_JupyterLab

### PR DESCRIPTION
**Pull request by steff456**
Fixes #59 

This PR,

- [x] Fixes the issues found in the Pitaya Smoothie and Github Light themes.

Here are some screenshots on how the current themes look,

**Github Light**

- [x] The active tab colour is noticeable
- [x] Hover background (menu) has enough contrast against the background colour
- [x] Colours in the kernel/tab manager sidebar 
- [x] Active indicator on the search bar
- [x] Toggle background is noticeable in the status bar

image

image

image

image

**Pitaya Smoothie**

- [x] Icons have enough contrast against the background
- [x] Active indicator on the search bar

image

### CODE REVIEW

**Review by gabalafou — COMMENTED** [review]
Great work, Stephannie! I understand that twiddling the CSS values is tricky, rinse-and-repeat kind of work! 

I just have one question. Does this PR fix all of the issues identified in the corresponding #59 or only some of them? I'm confused because some of the tasks in #59 are checked off but some are not.

**Comment by steff456** [comment]
This PR addresses all the issues in #59 for Github Light and Pitaya Smoothie. I made the changes of Github Dark in the PR #53.

Hope this clarifies!

**Comment by gabalafou** [comment]
Sure, thanks! Who is going to be responsible for checking off the tasks in issue #59?

Maybe also the title of the PR should be renamed to "Fixes for Pitaya Smoothie and Github Light." I understand that it says this already in the description, but right now the PR title is really generic.

**Review by gabalafou — APPROVED** [review]

**Comment by steff456** [comment]
> Who is going to be responsible for checking off the tasks in issue https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/59?

If we agree on the changes that I made for fixing them, I could check them once this PR is merged.

**Comment by trallard** [comment]
Disagree - changes should be checked BEFORE merging not the other way around

**Comment by trallard** [comment]
For example there is one that was not checked off for the dark theme as when I did the review I noticed the fix did not fix the colour contrast issue

**Comment by trallard** [comment]
Or you provided you tested / verified the changes and they meet the contrast requirements. Which IMO is part of doing the fix.

**Comment by steff456** [comment]
I added the screenshots to the description of this PR, I will update it with the contrast that every pair has. Hope that helps with the review!

**Comment by steff456** [comment]
With the latest changes, here are the screenshots of every change with its contrast check. (I already updated the issue as well)

# Github Light

- [x] The active tab colour is noticeable. 

Blue line and text have enough contrast against the background color
image

- [x] Hover background (menu) has enough contrast against the background colour

Gray background with white text
image

White background with gray hover color
image

- [x] Colours in the kernel/tab manager sidebar 

image

- [x] Active indicator on the search bar

image

image

- [x] Toggle background is noticeable in the status bar

image

Light gray background has enough contrast with toggle background
image

Toggle has enough contrast 
image

**Comment by trallard** [comment]
I am going to merge this as it seems ok after a quick review and I suppose users might raise issues later

**Pull request merged**
